.github/workflows/update_docker_versions.yaml

```diff
@@ -4,7 +4,7 @@ name: Update Pulumi Docker image versions
 # Run workflow on pushes to matching branches
 on:  # yamllint disable-line rule:truthy
   push:
-    branches: [python-migration]
+    branches: [develop]
```

Comment:
Do we need this to run on pushes if it is running every week?

Reply:
Probably not? Can you add a suggestion here that removes this?